src/Command/MemeCommand.php

```diff
@@ -0,0 +1,308 @@
+<?php
+namespace Drupal\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Console\Command\Command;
+
+/**
+ * Ported by Greg Anderson.
+ * Original code Copyright (C) 2013 by Muazzam Ali
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+class MemeCommand extends Command
+{
+    private $upperText;
+    private $lowerText;
+    private $color;
+    private $font;
+    private $im;
+    private $imgSize;
+
+    /**
+     * Generate a meme
+     *
+     * @command generate:meme
+     * @param string $topMessage The message to place at the top of the image.
+     * @param string $bottomMessage The message to place at the bottom of the image.
+     * @param string $image The path to the meme image to use.
+     * @default $image meme.png
+     * @option string $out The file to write the generated meme to.
+     * @default $out generated-meme.png
+     * @usage generate:meme "I don't always make memes" "But when I do, I use the command line" my-source-image.png
+     */
+    public function execute(InputInterface $input, OutputInterface $output)
```

Comment:
The configure() method is replaced with annotations on the execute() method. These annotations could also go on the class docblock; I'm not sure which is better.